generic-skiplink/skiplink.js

```diff
@@ -13,6 +13,7 @@ export const skiplink = `
     width: auto;
     margin: auto;
     opacity: 1;
+    pointer-events: auto;
```

Comment:
`pointer-events: auto` is missing `!important`.

Reply:
These styles are meant for usage inside a shadowRoot, so `!important` isnt needed here 🙂

Reply:
NVM I think I was looking at the wrong place.